### PR TITLE
prepare for v1.1.1 release

### PR DESCRIPTION
**Pull request by ianvernon**
Signed-off-by: Ian Vernon <ian@cilium.io>

---
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/4882)

### CODE REVIEW

**Comment by ianvernon** [comment]
test-me-please

File: AUTHORS [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
 off on commits in the Cilium repository:
 
+ackerman80                              arvind@covalent.io
 Alban Crequy                            alban@kinvolk.io
 Alexander Alemayhu                      alexander@alemayhu.com
 Alexei Starovoitov                      alexei.starovoitov@gmail.com
 Amey Bhide                              amey@covalent.io
 Amre Shakimov                           amre@covalent.io
-André Martins                           andre@cilium.io
+André Martins                          andre@cilium.io
```
Comment (aanm) [inline AUTHORS]: add a space here.

File: AUTHORS [hunk 2]
```diff
@@ -1,12 +1,13 @@
 The following people, in alphabetical order, have either authored or signed
 off on commits in the Cilium repository:
 
+ackerman80                              arvind@covalent.io
```
Comment (aanm) [inline AUTHORS]: Arvind already exists bellow

**Review by aanm — DISMISSED** [review]

File: AUTHORS (hunk 1 above)
Comment (tgraf) [inline AUTHORS]: I usually edited this manually to avoid this cosmetic problem.

**Review by tgraf — DISMISSED** [review]

File: AUTHORS (hunk 1 above)
Comment (ianvernon) [reply AUTHORS]: Fixed.

File: AUTHORS (hunk 2 above)
Comment (ianvernon) [reply AUTHORS]: Fixed.

File: AUTHORS (hunk 1 above)
Comment (ianvernon) [reply AUTHORS]: Fixed.

**Comment by ianvernon** [comment]
test-me-please

**Review by tgraf — APPROVED** [review]

**Review by aanm — APPROVED** [review]

**Comment by aanm** [comment]
test-missed-k8s

**Pull request merged**